resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/ezbake.spec.erb

```diff
@@ -19,19 +19,39 @@
 %global _app_data /opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>
 %global _projconfdir /etc/puppetlabs/<%= EZBake::Config[:real_name] %>
 
-# java 1.8.0 is available starting in fedora 20 and el 6
-%if 0%{?fedora} >= 20 || 0%{?rhel} >= 6
-%global open_jdk          java-1.8.0-openjdk-headless
-%else
-%global open_jdk          java-1.7.0-openjdk
-%endif
-
 %global rubylibdir        /opt/puppetlabs/puppet/lib/ruby/vendor_ruby
 
+%global _with_systemd  0
+%global _with_sysvinit 0
+%global _systemd_el    0
+%global _systemd_sles  0
+%global _old_el        0
+%global _sles          0
+
 %if 0%{?fedora} >= 18 || 0%{?rhel} >= 7
 %global _with_systemd 1
+%global _systemd_el   1
+%endif
+
+# We do not support any sles version less than 12, as there is not a sufficient
+# version of OpenJDK available to test against.
+%if 0%{?suse_version} >= 1210
```

Comment:
https://github.com/puppetlabs/ezbake/blob/master/resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/ezbake.spec.erb#L37

I pulled that line from the PE spec file. Does this mean that needs updating as well?